### ISSUE TITLE
Stops updating

### ISSUE THREAD

**Issue by jpmeijers** [PiotrMachowski/Home-Assistant-Lovelace-HTML-Jinja2-Template-card#22]
Ever since I added this card to my dashboard, the dashboard would stop refreshing data at some point. The graph widgets on the dashboard will draw lines up to "now", but with straight lines, like when there is no new data.

In the console I see an error:
![image](https://github.com/PiotrMachowski/Home-Assistant-Lovelace-HTML-Jinja2-Template-card/assets/948278/3350e3a3-4e8b-4563-9053-8e14647a53b3)

This makes me think the problem could be related to:
https://github.com/home-assistant/frontend/issues/13304
https://github.com/home-assistant/frontend/issues/11132

**Comment by PiotrMachowski** [comment]
> the dashboard would stop refreshing data at some point

What do you mean? Does it work for a few minutes and then crashes?

**Comment by jpmeijers** [comment]
Yes correct. Maybe more a couple of hours, then it stops refreshing/showing new data. Next time it happens I'll have a screenshot to explain better.

I experience this on an Android tablet, running the Home Assistant app. Therefore debugging this is very difficult. I'm now leaving the same dashboard open in my Chrome browser, with the dev tools console running, hoping that I can catch any other errors than the one mentioned above.
